proto/osmosis/tokenfactory/v1beta1/query.proto

```diff
@@ -33,12 +33,12 @@ service Query {
         "/osmosis/tokenfactory/v1beta1/denoms_from_creator/{creator}";
   }
 
-    // DenomsFromCreator defines a gRPC query method for fetching all
+  // DenomsFromCreator defines a gRPC query method for fetching all
   // denominations created by a specific admin/creator.
-  rpc DenomBeforeSendHook(QueryDenomBeforeSendHookRequest)
-      returns (QueryDenomBeforeSendHookResponse) {
+  rpc DenomBeforeSendListener(QueryDenomBeforeSendListenerRequest)
+      returns (QueryDenomBeforeSendListenerResponse) {
     option (google.api.http).get =
-    "/osmosis/tokenfactory/v1beta1/denoms/{denom}/before_send_hook";
+        "/osmosis/tokenfactory/v1beta1/denoms/{denom}/before_send_hook";
```

Comment:
It seems we are calling this a "before send listener" but it retains the "before_send_hook" nomenclature. Is this intentional?

Reply:
not intentional 🥲